plugin/al/usdmayautils/AL/usdmaya/utils/MeshUtils.cpp

```diff
@@ -2007,7 +2109,7 @@ void MeshExportContext::copyNormalData(UsdTimeCode time, bool copyAsPrimvar)
                   missing[normalIndices[i]] = faceConnects[i];
                 }
                 else
-                if(it->second != uint32_t(faceConnects[i]))
+                if(it->second != faceConnects[i])
```

Comment:
error: comparison between signed and unsigned integer expressions [-Werror=sign-compare]
`if(it->second != faceConnects[i])`